main/py-pbr/APKBUILD

```diff
@@ -7,24 +7,49 @@ pkgrel=0
 pkgdesc="Python Build Reasonableness"
 url="https://pypi.python.org/pypi/pbr"
 arch="noarch"
-license="APACHE2.0"
-depends="python2"
-depends_dev=""
-makedepends="$depends_dev python2-dev py-setuptools"
-install=""
-subpackages="$pkgname-dev"
+license="ASL 2.0"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
 
-builddir="$srcdir/$_pkgname-$pkgver"
+builddir="$srcdir"/$_pkgname-$pkgver
+prepare() {
+	local i
+	cd "$builddir"
+	for i in $source; do
+		case $i in
+		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+		esac
+	done
+}
```

Comment:
Applying of patches was added into the default `prepare()` function some time ago, see [the code](https://github.com/alpinelinux/abuild/blob/69fd2290feaf7b8ae3d5cbaa2d1f6dda4915d71d/abuild.in#L566-L584). We’re cleaning abuilds as part of update of each package, as you can see in many recent commits. The same applies to `_builddir` → `builddir`.